gremlin-python/src/main/python/tests/process/test_translator.py

```diff
@@ -373,6 +374,30 @@ class SuperStr(str):
         tests.append([g.V().has("p1", None),
                      "g.V().has('p1',null)"])
 
+        # 104
+        vertex = Vertex(0, "person")

```

Comment:
Could you add similar test for `vertex = Vertex(0, "'person'")` to confirm correct work with strings?

Reply:
Hmm, ok. Is that a use case to crate a label (that's a string) with a single quote? Or it's just to test?

Reply:
create label with single quote to verify serialization

Reply:
Updated to using repr() for string formatting for now.